ansible/roles/heritage-frontend-base/vars/main.yml

```diff
@@ -5,8 +5,19 @@ users:
 
 user_home_path: /home/ewf
 
-s3_files:
-  - packages/oracle/oracle-instantclient11.2-basic-11.2.0.2.0.x86_64.rpm
-  - packages/oracle/oracle-instantclient11.2-devel-11.2.0.2.0.x86_64.rpm
-  - packages/oracle/oracle-instantclient11.2-sqlplus-11.2.0.2.0.x86_64.rpm
-  - application/ewf/frontend/ewf-frontend.tar.gz
+tar_files:
+  - ewf-frontend-app.tar.gz
+  - ewf-frontend-static.tar.gz 
+
+s3_file_app_static: /application/ewf/frontend/ewf-frontend-static.tar.gz
+s3_file_app: /application/ewf/frontend/ewf-frontend-app.tar.gz
+s3_files_app_static_name: ewf-frontend-static.tar.gz
+s3_files_app_name: ewf-frontend-app.tar.gz
```

Comment:
Should these be be in defaults.yml, you have defined them in host_vars/ as well which overrides vars and defaults in a role.
Lets makes sure we dont define variables in too many places or in different places between roles.

Reply:
The vars have been removed from the role var to defaults and host_vars removed.